File: Desafios em C#/3040 - A Árvore de Natal.cs
using System;

//Developed by: @LucasMarcuzo

namespace _3040___A_Árvore_de_Natal
{
    class Program
    {
        static void Main(string[] args)
        {

            int ts = int.Parse(Console.ReadLine());

            for (int i = 0; i < ts; i++)
            {
                string[] v = Console.ReadLine().Split();

                int altura = int.Parse(v[0]);
                int diametro = int.Parse(v[1]);
                int galhos = int.Parse(v[2]);

                if(altura >= 200 && altura <= 300)
                {
                    if (diametro >= 50)
                    {
                        if(galhos > 150)
                        {
                            Console.WriteLine("Sim");
                        }
                        else
                        {
                            Console.WriteLine("Nao");
                        }
                    }
                    else
                    {
                        Console.WriteLine("Nao");
                    }
                }
                else
                {
                    Console.WriteLine("Nao");
                }
            }
        }
    }
}

File: Desafios em JavaScript/3040 - A Árvore de Natal.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var ts = parseInt(lines.shift());

for (let i = 0; i < ts; i++)
{
    var ln = lines.shift().split(" ");

    var altura = parseInt(ln.shift());
    var diametro = parseInt(ln.shift());
    var galhos = parseInt(ln.shift());

    if(altura >= 200 && altura <= 300)
    {
        if (diametro >= 50)
        {
            if(galhos > 150)
            {
                console.log("Sim");
            }
            else
            {
                console.log("Nao");
            }
        }
        else
        {
            console.log("Nao");
        }
    }
    else
    {
        console.log("Nao");
    }
}